### PR TITLE
MAUI-1646_UG for plot area background support

### PR DESCRIPTION
**Pull request by SaiyathAliFathima**
### Feature description ###
Preparing UG document for plot area background view support for the circular and cartesian charts.

### Analysis and design ###
Referred the requirement document of plot area background view  [https://syncfusion.sharepoint.com/:w:/s/MAUI/EZwzbXIW8r1MjUxQfsQBQHMBTVJgPTl0oLFpQCix_nshDg?e=hU82P4](url)

### Solution description ###
This UG document helps to guide the user to understand the feature.

### 
plot_view

plot_view

### Areas affected and ensured ###
None

### API Changes ###
None

### Behavioral Changes ###
None

### Test cases ###
Not Applicable

### Testbed sample location ###
Not Applicable

### Does it have any known issues?
None

### Does it have memory leak?
None

### CODE REVIEW

**Comment by SaiyathAliFathima** [comment]
rebuild

File: MAUI/Cartesian-Charts/Plot-Area-Background-View.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+{% tabs %}
+
+{% highlight xaml %}
+
+<chart:SfCartesianChart>
+   <chart:SfCartesianChart.PlotAreaBackgroundView>
+      <AbsoluteLayout>
+         <Label Text="Copyright @ 2022 Parker Industries"
+                FontSize="18"
+                AbsoluteLayout.LayoutBounds="1,1,310,33"
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Plot-Area-Background-View.md:32]: layout bounds -1, -1 will not be any impact in position promotion.

File: maui-toc.html [hunk 2]
```diff
@@ -149,6 +149,7 @@
                     <li><a href="/maui/cartesian-charts/scatter">Scatter Chart</a></li>
                 </ul>
             </li>
+            <li><a href="/maui/cartesian-charts/plot-area-background-view">Appearance</a></li>
```
Comment (Saravanan-Madhesh) [inline maui-toc.html:152]: File name "Appearance.md"

File: MAUI/Cartesian-Charts/Plot-Area-Background-View.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: Plot Area Background View  in .NET MAUI Chart control | Syncfusion
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Plot-Area-Background-View.md:3]: Appearance in .NET MAUI Chart control | Syncfusion

File: MAUI/Cartesian-Charts/Plot-Area-Background-View.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: Plot Area Background View  in .NET MAUI Chart control | Syncfusion
+description: This section explains about how to configure the Plot Area Backgrund and its features in .NET MAUI Chart (SfCartesianChart).
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Plot-Area-Background-View.md:4]: Learn here all about appearance customization in .NET MAUI Chart (SfCartesianChart), its elements and more.

File: MAUI/Circular-Charts/Plot-Area-Background-View.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: Plot Area Background View  in .NET MAUI Chart control | Syncfusion
+description: This section explains about how to configure the Plot Area Backgrund and its features in .NET MAUI Chart (SfCircularChart).
+platform: maui
```
Comment (Saravanan-Madhesh) [inline MAUI/Circular-Charts/Plot-Area-Background-View.md:5]: change title, file name and description like cartesian

File: MAUI/Circular-Charts/Plot-Area-Background-View.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+                Rotation="340"
+                FontSize="80"
+                FontAttributes="Bold,Italic"
+                TextColor="Gray"
+                Margin="10,0,0,0"
+                AbsoluteLayout.LayoutBounds="0.5,0.5,-1,-1"
+                AbsoluteLayout.LayoutFlags="PositionProportional"
+                Opacity="0.3" />
+    </AbsoluteLayout>
+   </chart:SfCircularChart.PlotAreaBackgroundView>
```
Comment (Saravanan-Madhesh) [inline MAUI/Circular-Charts/Plot-Area-Background-View.md:38]: image has border, no code how it achieve.  simplify the label code like below in single line. 

`  <Label Text="Copyright @ 2001 - 2022 Syncfusion Inc"AbsoluteLayout.LayoutBounds="1,1,-1,-1" AbsoluteLayout.LayoutFlags="PositionProportional" Opacity="0.4" /> `

`<Label Text="CONFIDENTIAL" Rotation="340" AbsoluteLayout.LayoutBounds="0.5,0.5,-1,-1" AbsoluteLayout.LayoutFlags="PositionProportional" Opacity="0.3" />`

File: MAUI/Circular-Charts/Plot-Area-Background-View.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+absoluteLayout.Children.Add(copyRight);
+var watermark = new Label()
+{
+   Text = "CONFIDENTIAL",
+   Rotation = 340,
+   FontSize = 80,
+   FontAttributes = FontAttributes.Bold,
+   TextColor = Colors.Gray,
+   Opacity = 0.3,
+};
```
Comment (Saravanan-Madhesh) [inline MAUI/Circular-Charts/Plot-Area-Background-View.md:59]: Make code changes in single line like xml.

**Review by Saravanan-Madhesh — CHANGES_REQUESTED** [review]

File: maui-toc.html (hunk 2 above)
Comment (SaiyathAliFathima) [reply maui-toc.html]: I have changed the file name

File: MAUI/Cartesian-Charts/Plot-Area-Background-View.md (hunk 3 above)
Comment (SaiyathAliFathima) [reply MAUI/Cartesian-Charts/Plot-Area-Background-View.md]: I have changed the title content.

File: MAUI/Cartesian-Charts/Plot-Area-Background-View.md (hunk 4 above)
Comment (SaiyathAliFathima) [reply MAUI/Cartesian-Charts/Plot-Area-Background-View.md]: I have changed the description content.

File: MAUI/Circular-Charts/Plot-Area-Background-View.md (hunk 5 above)
Comment (SaiyathAliFathima) [reply MAUI/Circular-Charts/Plot-Area-Background-View.md]: I have changed the file name and title, description content.

File: MAUI/Circular-Charts/Plot-Area-Background-View.md (hunk 7 above)
Comment (SaiyathAliFathima) [reply MAUI/Circular-Charts/Plot-Area-Background-View.md]: I have updated the code.

**Review by Saravanan-Madhesh — APPROVED** [review]
The changes are fine @SyncfusionKarthikeyan please update your concern

**Pull request merged**